src/gdcmodels/esmodels/models.py

```diff
@@ -2,6 +2,19 @@
 
 from typing_extensions import NotRequired, Protocol, TypedDict
 
+GRAPH_INDICES = tuple(("gdc_from_graph", d) for d in ("annotation", "case", "file", "project"))
+VIZ_INDICES = tuple(
+    (i, i)
```

Comment:
Is the VIZ_INDICES tuple structure so that it kind of matches the footprint of the GRAPH_INDICES structure?

Reply:
Yes and that is the current state of things. It lends to a standard modeling of the results of `get_es_models` as well i.e. `index/doc_type` which for the viz indices is just a repeat.